babelTransform.js

```diff
@@ -0,0 +1,18 @@
+const babelJest = require("babel-jest").default;
+
+module.exports = {
+  process(sourceText, sourcePath, options) {
+    const alias = options?.config?.moduleNameMapper;
+    const extensions = options.config.moduleFileExtensions;
+    const babelTransformer = babelJest.createTransformer({
+      presets: [["@babel/preset-react"], ["@babel/preset-env"], ["@babel/preset-typescript"]],
+      plugins: [
+        ["transform-barrels", { executorName: "jest", alias: alias, extensions: extensions, isCacheEnabled: false }],
+      ], // set to true when all else worked
```

Comment:
I don't understand this comment; what does it mean?

Reply:
I've found out about the isCacheEnabled parameter, and wanted to leave me a note here to try to enable it when I fixed the other problems. During testing, caching was not helping at all, since it wouldn't delete the cache when you've made changes to the configuration. 
---
I did try this parameter in the meantime though and it didn't yield any performance gains, but would save some temporary files. So we shouldn't enable this. Jest-Caching, enabled by default, on the other hand is giving quite a boost, and would probably also circumvent any plugin caching.